lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ if(CMAKE_UFE_V2_FEATURES_AVAILABLE)
             UsdContextOpsHandler.cpp
             UsdObject3d.cpp
             UsdObject3dHandler.cpp
+            UsdSetXformOpUndoableCommandBase.cpp
```

Comment:
Went back and forth on whether I added my new base class to the existing UsdTransform3dUndoableCommands.{h,cpp}, or create a new file.  Ended up deciding that a base class was worth its own file, but this is debatable.